Name: No Refresh In Browser
Version: 1.3.5-1
Author: Fritos1406
Description: No refresh in the browser when away from it

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/widget_webview.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/widget_webview.js
@@ -537,7 +537,7 @@
 			// This browser page was purged when the application was not in the foreground.
 			// Now that we're active again let's reconnect and reload the page.
 			Mojo.Log.info("Reconnecting adapter to server.");
-			this._connectAdapterToServer();
+			this._connectAdapterToServer(false);
 		}
 	},
 
